state/sqlserver/sqlserver.go

```diff
@@ -478,9 +450,16 @@ type dbExecutor interface {
 }
 
 func (s *SQLServer) executeSet(db dbExecutor, req *state.SetRequest) error {
-	json, err := json.Marshal(req.Value)
-	if err != nil {
-		return err
+	var err error
```

Comment:
is this same behavior of redis component?

Reply:
Yes.  Other components may have a similar issue as well when handing data received by the grpc api.